ci-scripts/JenkinsFile-OAI-Container-GitHub

```diff
@@ -295,12 +295,11 @@ pipeline {
                     } catch (Exception e) {
                       echo 'OK if not present'
                     }
-                    // In case of the full build, we need the host RHEL8 credentials.
-                    if (params.REGRESSION_TEST) {
-                      sh('mkdir -p tmp/ca tmp/entitlement')
-                      sh('cp /etc/pki/entitlement/*pem tmp/entitlement')
-                      sh('sudo cp /etc/rhsm/ca/redhat-uep.pem tmp/ca')
-                    }
+                    // Copying the host RHEL8 credentials all the time
+                    sh('mkdir -p ./etc-pki-entitlement ./rhsm-conf ./rhsm-ca')
+                    sh('cp /etc/pki/entitlement/*pem ./etc-pki-entitlement')
```

Comment:
NIT^2: `*pem` -> `*.pem`? or are there actually files/folders ending in `pem` without `.`?